src/main/java/Itstime/planear/schedule/Controller/ScheduleController.java

```diff
@@ -0,0 +1,30 @@
+package Itstime.planear.schedule.Controller;
+
+import Itstime.planear.common.ApiResponse;
+import Itstime.planear.exception.PlanearException;
+import Itstime.planear.schedule.DTO.ScheduleRequestDTO;
+import Itstime.planear.schedule.DTO.ScheduleResponseDTO;
+import Itstime.planear.schedule.Service.ScheduleService;
+import jakarta.servlet.http.HttpSession;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+public class ScheduleController {
+    private final ScheduleService scheduleService;
+
+    @PostMapping("/schedule")
+    public ApiResponse<ScheduleResponseDTO.ScheduleCreateDTO> create(
+            @RequestBody ScheduleRequestDTO.ScheduleCreateDTO scheduleCreateDTO,
+            @RequestHeader("memberId") Long memberId) throws PlanearException {
```

Comment:
```suggestion
            @RequestHeader("user-no", required=false) Long memberId)  {
```
이렇게 줘야 제대로 받을 수 있을껄
우리 헤더 규칙이 user-no 라서 memberId 로 받으면 안돼

추가로 throws 안 붙혀도 돼 runtimeException 을 상속해서